### ISSUE TITLE
Doesn't print large dataset in output

### ISSUE THREAD

**Issue by TwiceII** [BetterThanTomorrow/calva#97]
Whenever I try to eval expression that results in (fairly) large data, Calva output just does nothing.
`Evaluating:`
`(range 0 2000)`

Is there a limit of lines that Calva can print? I can't examine large maps and sequences in my repl.

**Comment by PEZ** [comment]
Hi! Yes, unfortunately there is a limit to the inline evaluation. It's quite arbritrary, because it is rather a bug than anything else.

A workaround is to evaluate it in the REPL terminal. Default keybindings.

* `ctrl+alt+v alt+e` -> Evaluate current form.
* `ctrl+alt+v alt+space` -> Evaluate current top level form (also works inside `comment` forms).

I am sorry for the inconvenience!

We (well, mostly @mseddon) are working on some much better ways to examine large datastructures in Calva. It involves [some serious ground work](https://github.com/BetterThanTomorrow/repl-interactor), though, so might take a little while before we can reap the *large datastructures* fruits of that work.

**Comment by TwiceII** [comment]
Thanks! Seems like you both doing a really cool thing, I suppose you're already familiar with [REBL](https://github.com/cognitect-labs/REBL-distro)? Thought that could be a great source of motivation. Anyway, I guess we can close this issue.

**Comment by mseddon** [comment]
Yep, and we are *definitely* looking towards getting a nice REBL experience into Calva in the future. Although there are a few more 'must have' features to get us up to parity with Cider before we chase those rainbows. ;)

**Comment by PEZ** [comment]
I'd like to keep the issue open, so that it easier to find for someone else running into this limitation. Also serves as a reminder to myself that I should investigate the issue closer. We will need to have it fixed to be able to feed the new REPL UI without this strange limit anyway.

**Comment by mseddon** [comment]
@PEZ This is caused by the old nREPL client dropping the response with long input isn't it? The currently shipped new client should not have this problem anymore. Vote to close?

**Comment by PEZ** [comment]
Yes, this is now fixed.

What now became an issue is that inline content gets truncated from the
end. But that could be a separate issue.
